CMakeLists.txt

```diff
@@ -389,6 +389,13 @@ if(WITH_DPDK)
   endif()
 endif()
 
+option(WITH_LIBRADOSSTRIPER "Build with libradosstriper support." ON)
+if(WITH_LIBRADOSSTRIPER)
+  message(STATUS "Enabling libradosstriper in \"rados\" tool.")
```

Comment:
@chardan Could you add "support" after "libradosstriper" in this line?

Reply:
I sure can!

Reply:
This doesn’t appear to have happened.

Reply:
Ah, now I see :-) @chardan Please add the word "support" as requested above

Reply:
Yeah, the threading display on my review is a bit weird.

I’m indifferent to the wording but noticed it was discussed and not done. :)